README.md

```diff
@@ -331,6 +342,7 @@ For Vert.x version 2 check [this page](./vert-x2.md).
 - [User Group](https://groups.google.com/forum/?fromgroups#!forum/vertx) - Discuss all user issues related to *using* Vert.x.
 - [Developer Group](https://groups.google.com/forum/?fromgroups#!forum/vertx-dev) - A group for Vert.x core *developers* and *contributors*.
 - [IRC channel](https://webchat.freenode.net/?channels=#vertx) - This is our day-to-day office: #vertx on freenode.net.
+- [Glitter chat](https://gitter.im/eclipse-vertx/vertx-users) general chat for Vert.x releated chat.
```

Comment:
typo : Gitter
can you also say that the IRC channel is replaced by the Gitter chat ?

Reply:
actually you can remove the IRC channel